documentation/source/swagger/swagger_products_services_apis.yaml

```diff
@@ -2603,7 +2603,7 @@ components:
       properties:
         cnpjNumber:
           type: string
-          pattern: '^(\d{14})$'
+          pattern: '\d{14}|^NA$'
```

Comment:
o caracter ^ indica início da string e o caracter $ indica o fim, portanto o correto seria: '^(\d{14})$|^NA$'